src/cobalt.rs

```diff
@@ -450,3 +464,37 @@ fn create_jsonfeed(
 
     Ok(())
 }
+fn create_sitemap(
+    path: &str,
+    dest: &path::Path,
+    collection: &Collection,
+    documents: &[Document],
+    collection_pages: &Collection,
+    documents_pages: &[Document],
+) -> Result<()> {
+    let sitemap_path = dest.join(path);
```

Comment:
From an API perspective (even if internal) I think I'd prefer this to be in the caller